scavenger-entity/src/main/kotlin/com/navercorp/scavenger/entity/ExportSnapshotMethodEntity.kt

```diff
@@ -0,0 +1,37 @@
+package com.navercorp.scavenger.entity
+
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Table
+
+@Table("snapshots")
+data class ExportSnapshotMethodEntity(
```

Comment:
We define Entity per each table.
Here the ExportSnapshotMethodEntity is not the table but projection for join result.
I highly recommend renaming this into ExportSnapshotMethodDbRow(~DbRow is our naming convention for projection) and  moving this into 
/scavenger-api/src/main/kotlin/.../entity/ExportSnapshotMethodDbRow.kt

Reply:
I fixed its name from Entity to DbRow. But there was no 'entity' directory under 'scavenger-api' and I originally made it under 'scavenger-entity'. So, I just rename the file and didn't move. Did I modify it correctly..?
I tried to fix this on commit a8be219.

Reply:
just make folder.... under scavenger-api
so far.. we did not need it.
but i think it is the time to make it.

Reply:
I moved the file on commit a9fd260.